data/scripts.js

```diff
@@ -108,7 +108,7 @@ let BattleScripts = {
 			}
 			// Dancer activates in order of lowest speed stat to highest
 			// Ties go to whichever Pokemon has had the ability for the least amount of time
-			dancers.sort(function (a, b) { return -(b.stats['spe'] - a.stats['spe']) || b.abilityOrder - a.abilityOrder; });
+			dancers.sort(function (a, b) { return -(b.storedStats['spe'] - a.storedStats['spe']) || b.abilityOrder - a.abilityOrder; });
```

Comment:
We cache a `.speed` which might be the actual thing to do here. @Marty-D?

Reply:
Replacing with `.speed` has the Dancer test for this behaviour fail, so probably not.

Reply:
Sounds like Dancer to me. Although I wish the comment would make this clear.

Also what is with this incredibly outdated code standard anyway?

Reply:
```suggestion
			dancers.sort((a, b) =>
				-(b.storedStats['spe'] - a.storedStats['spe']) || b.abilityOrder - a.abilityOrder
			);
```